setup.md

```diff
@@ -0,0 +1,57 @@
+Setup 
+==========
+
+Requirements
+---------------
+
+	# Python
+	- Installation Guide: https://www.python.org/about/gettingstarted/
+
+	# Superset
+	- Installation & Configuration: https://superset.incubator.apache.org/installation.html
+	- Introduction Video: https://youtu.be/3Txm_nj_R7M
+
+Making your own build
+----------------------
+
+	#------------------------------------------------------------------
+	# Preparing the Database
+	#------------------------------------------------------------------
+
+	- Superset imports data from databases, thereby the data has to be consolidated to this form. 
```

Comment:
mention "From csv to a database, we choose sqlite blah blah " and remove the indentation. Keep the indentation for the line of codes or special instructions like the sqlite db link format